packages/cozy-doctypes/src/File.js

```diff
@@ -199,7 +199,7 @@ class CozyFile extends Document {
         )
       }
     } catch (error) {
-      if (/Not found/.test(error)) {
+      if (/Not Found/.test(error.message)) {
```

Comment:
Can you add a test ? Seems like this should be extracted from the method. Thanks !

Reply:
Trying to figure out why in this case I have an object and a case issue (since I already have a failing test... testing `Not found`)